src/main/java/tel/schich/javacan/BcmTimeval.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * The MIT License
+ * Copyright © 2018 Phillip Schichtel
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package tel.schich.javacan;
+
+import java.time.Duration;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Value;
+
+/**
+ * Java representation of a {@code bcm_timeval} from native {@code linux/can/bcm.h}
+ *
+ * @see https://www.kernel.org/doc/html/latest/networking/can.html#broadcast-manager-receive-filter-timers
+ */
+@Value
+@AllArgsConstructor
+@Builder
+public class BcmTimeval {
```

Comment:
Is there actually any benefit of this class instead if directly using Duration?

Reply:
The only hairsplitting benefit of this class is that the documentation states that the minimum time resolution is micro-seconds, not nano-seconds as the Duration.
When i think about it... it is not worth it... we are better of if it is gone.

Reply:
The kernel also often doesn't fully support the resolution that some time parameters accept. A note in the documentation about the maximum supported resolution and what happens to higher resolution information (truncation, rounding, ...) should be enough.
So if the precision is the only point, then yes please remove it :)

Reply:
sorry. i forgot to push my changes.
I already did this.